build

```diff
@@ -26,13 +28,8 @@ tagopt=(
     --tag "${REGISTRY}/${NAME}:${VERSION}${PKG}"
 )
 
-# Include the git tag if differnt from the version.
-if [ "$GITHUB_REF_TYPE" == "tag" ]; then
-    GIT_TAG=${GITHUB_REF/refs\/tags\//}
-    if [ "$GIT_TAG" != "$VERSION" ]; then
-        tagopt+=(--tag "${REGISTRY}/${NAME}:${GIT_TAG}${PKG}")
-    fi
-fi
```

Comment:
Hard for me to see what's going on with this diff — do we want to keep this?

Reply:
It's not useful to us, though it doesn't hurt to keep around to minimize the diff